packages/core/src/command/consolidated/consolidated-filter.ts

```diff
@@ -2,7 +2,7 @@ import { Bundle, Resource } from "@medplum/fhirtypes";
 import { ResourceTypeForConsolidation, SearchSetBundle } from "@metriport/shared/medical";
 import {
   buildBundleEntry,
-  getReferencesFromResources,
+  getReferencesAndMissingOnes,
```

Comment:
This is a bit oddly named.

Reply:
Yeah, I agree. This started as a refactoring but I ended up building a dedicated function (`getPatientReferencesFromResources`).

I'll revert this and leave the original name.

Reply:
done